hwilib/devices/jade.py

```diff
@@ -91,6 +92,7 @@ def func(*args: Any, **kwargs: Any) -> Any:
 
 # This class extends the HardwareWalletClient for Blockstream Jade specific things
 class JadeClient(HardwareWalletClient):
+    MIN_SUPPORTED_FW_VERSION = semver.VersionInfo(0, 1, 32)
```

Comment:
Right - HWI hasn't been released with Jade support yet afaik - and fw 0.1.32 is the first to support sortedmulti() - and it seemed cleaner to just enforce than min version across the board at this early stage (rather than check different min versions depending on what you were trying to do).

Reply:
> @JamieDriver but when I manually try to craft the descriptor, Jade seems to think I want to override the previously registered wallet.

We derive a name to use to register the multisig, based on the signers, threshold and script-type.  The multisig records are keyed in Jade by this name, so if the threshold, signers, type are unchanged it'll end up with the same name and so overwrite the existing.

Another multisig with different signers, threshold, or script-type should generate a different name, and so will store another multisig registration.  Atm we allow up to 8 - they can be viewed/deleted on the 'Settings' screen.

Reply:
This happened to me multiple times on the same wallet though (I only have on multisig test wallet with Jade). Same signers, threshold and script type.

Reply:
I'm not sure I am understanding properly.  ;-)
Same signers/threshold/script-type should re-use the existing multisig registration record.  Trying to change the order of the signers (or changing bip67 sorting flag) will require reconfirming on the hww.

* First use will require a registration/confirmation on the device.
* Using same descriptor, but next sequence/index should be fine and not require any registration/confirmation of the multisig.
* If I change the order of the signers, or change to/from a sorted-multi(), (but keep same threshold, signers and script-type) it will want to re-confirm, and will overwrite the existing registration record.
* If I change the script-type, threshold or the signers, it will want to create a new (ie. second/separate) registration.

(*) NOTE: trying to register totally identical details as that already held is a no-op and just returns immediately without needing to eyeball/confirm on the hww.

If you think you are seeing something different, can you provide details and I will investigate ?  Cheers.

Reply:
I created a sorted_multi wallet with the Specter PR. I used Specter and Core and a manual HWI call to display the address, as well as to sign transactions. Pretty much every time I got the "do you want to replace the wallet" prompt on the device. So it seems to think the wallet changed. Based on what you're describing that shouldn't be happening.

Reply:
"do you want to replace the wallet" ?  On Jade ?

If the Jade registration details had changed it would make you scroll through the multisig summary screen, a screen per signer, and on the final confirm screen show a red message: "Warning: overwriting existing registration".
Is that what you mean ?

If so - we'd have to enable logging on the python client side and see what messages are being passed to the Jade.
eg. it may be that the order of the xpubs keeps changing in the descriptor info.

Reply:
I suspect - but haven't thoroughly tested -  that the use of `sorted_multi` caused the xpub's to get swapped, depending on whether the wallet descriptors were used directly or inferred from the script. See bitcoin/bitcoin#24003.